source/Coop.Core/Templates/ServiceTemplate/Messages/NetworkMessageTemplate.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+
+namespace Coop.Core.Templates.ServiceTemplate.Messages
+{
+    [ProtoContract]
+    public record NetworkMessageTemplate : IEvent
+    {
```

Comment:
Perhaps add a ProtoMember or two so that people new people simply can see how to add contents